org.freedesktop.Sdk.Extension.nvidia-base.metainfo.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>org.freedesktop.Sdk.Extension.nvidia-base</id>
+  <extends>org.freedesktop.Sdk</extends>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <name>nvidia-base</name>
+  <summary>Package NVIDIA drivers faster</summary>
+  <url type="homepage">https://github.com/flathub/org.freedesktop.Sdk.Extension.nvidia-base</url>
+  <description>
+    <p>
+      Base extension with pre-built static libraries to be used by the NVIDIA driver extension for Flatpak (https://github.com/flathub/org.freedesktop.Platform.GL.nvidia) to speed up packaging time.
+    </p>
+  </description>
+  <releases>
```

Comment:
Release doesn't make sense here, I'd drop this it's not mandatory for extensions.

Reply:
Great, I didn't know that. Removed it as well.